extensions/ikea-idasen/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "ikea-idasen",
+  "title": "Ikea Idasen",
+  "description": "Let's you control your IKEA IDASEN from within Raycast",
+  "icon": "command-icon.png",
+  "author": "antoni",
+  "categories": [
+    "Productivity",
+    "Other"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Table Control",
+      "subtitle": "Ikea Idasen",
+      "description": "Control your Ikea Idasen within raycast",
+      "mode": "view"
+    },
```

Comment:
Does it make sense to keep this now that they both have their own command?

Reply:
There is another command, which only will be shown when you not have installed the app, which you can download instantly. Later I also plan to add an option to add a custom high.

Reply:
I added the option to set a custom hight